Makefile.am

```diff
@@ -42,6 +42,7 @@ TOR_UTIL_LIBS = \
 	src/common/libor.a \
         src/lib/libtor-log.a \
         src/lib/libtor-lock.a \
+        src/lib/libtor-fdio.a \
```

Comment:
Quick question: Is the goal of this library to also get the `tor_open_*` family of functions (currently mostly in `compat.c`)?

If yes, I'm wondering if `libtor-io.a` wouldn't be better and we could also bring in all the network low level functions? Or for that you had in mind a `libtor-network.` kind of separation?

Reply:
No, tor_open_* needs to be at a higher level, or else it will cause a circular dependency.  Those functions have a lot of complexity, and have the potential to log, and require the sandbox code.